include/mm/pmm.h

```diff
@@ -52,6 +52,8 @@ extern unsigned long __start_bss_init[], __end_bss_init[];
 
 extern unsigned long __start_rmode[], __end_rmode[];
 
+extern struct ktf_param __start_cmdline_init, __end_cmdline_init;
```

Comment:
IIRC this should be declared as `extern struct ktf_param __start_cmdline_init[], __end_cmdline_init[];`